Bumbodium/Views/Shared/Components/AvailabilityComponent.razor

```diff
@@ -58,7 +58,7 @@
                         EndDateTime = args.End,
                         Type = SelectedType,
                         // TODO replace with actual id
-                        EmployeeId = 1
+                        EmployeeId = "1"
```

Comment:
this breaks the availability code during testing. The new seed data employee's id is "b74ddd14-6340-4840-95c2-db12554843e5" instead of 1 causing this to fail due to a foreign key constraint